appendix-basic65.tex

```diff
@@ -9430,9 +9441,11 @@ \subsection{WINDOW}
                  depending on the screen mode.
 
 \item [Remarks:] There can be only one window on the screen.
-                 Striking the HOME key twice or printing
+                 Pressing the HOME key twice or printing
                  CHR\$(19)CHR\$(19) will reset the window
                  to the default full screen.
+                 Refer to the {\bf CHR\$} command on page \pageref{chrcommand}
```

Comment:
Hmm, on this occasion, maybe the extra 'refer to' isn't needed, as the earlier remark provides the exact CHR$() value needed, and I don't the details in the CHR$ command will provide much extra value to the reader on this occasion.

Reply:
good point, i'll take it out